src/sentry/utils/os_normalization.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+import re
+
+# Environment.OSVersion (GetVersionEx) or RuntimeInformation.OSDescription, on Windows
+windows = re.compile('^(Microsoft )?Windows (NT )?(?P<version>\d+\.\d+\.\d+).*$')
+# Environment.OSVersion or RuntimeInformation.OSDescription (uname)
+# on Mono and CoreCLR on macOS, iOS, Linux, etc
+uname = re.compile('^(?P<name>[a-zA-Z]+) (?P<version>\d+\.\d+\.\d+(\.[1-9]+)?).*$')
```

Comment:
nit: Most regexes in sentry have this naming style `_windows_re`, `_uname_re`

Reply:
I would add `(?i)` to make it case insensitive (the windows one)